.ci/pipeline/release_jjb.yaml

```diff
@@ -70,6 +70,7 @@
             keep-system-variables: true
             properties-content: |
               jjb_proj={jjb_proj}-dr
+              WEBREPO_URL=webrepo.mtr.labs.mlnx
```

Comment:
Are you sure we dont need this for prod job definition as well? did you tested both ?